modules/ROOT/pages/rtf-scale.adoc

```diff
@@ -61,11 +61,10 @@ Aspects to consider:
 [%header%autowidth.spread]
 |===
 | Cluster Recommendations | Default | 200 Node/8000 App Resource Settings 
-| Agent Resources | Memory 200 mb request, 500 mb limit. CPU 100m request, 1vCore limit. | 3 GB (Memory)/2vCore (request == limit)  
```

Comment:
@kmendiratta-crm @jiahongli18 should we keep the old values as well? 
For older version customers will still need old values, right? 
Should we provide benchmarking based on rtf-agent version ?

Reply:
As discussed on slack, old will show in the <=2.6 tabs